tools/python/boutiques/templates/template_docopt_descriptor.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
As discussed, potentially use `bosh create` with no arguments to generate a template from which you can build?